models/Events.js

```diff
@@ -0,0 +1,87 @@
+const { Schema, model } = require('mongoose');
+
+const eventsSchema = new Schema({
+  name: {
+    type: String,
+    required: [true, 'The name of Event is required'],
+    trim: true
+  },
+  description: {
+    type: String,
+    required: [true, 'The description is required'],
+    trim: true
+  },
+  talks: [{
+    name: String,
+    description: String,
+    date: Date,
+    hour: Date,
+    speakers: [{
+      name: {
+        type:String,
+        require: true,
+        trim: true
+      },
+      avatar: String,
+      twitter: String,
+      biografia: String,
+      rol: String
+    }]
+  }],
+  eventStatus: {
+    type: Boolean,
+    default: false
+  },
+  date: Date,

```

Comment:
Date & hour can be in the same attribute instead of two

Can use 2 field to Initial Date & End Date
iniDate: DateTime
endDate: DateTime